shared/middleware/helpers/auth_helpers.rb

```diff
@@ -69,15 +69,15 @@ def has_permission?(permission)
 end
 
 # @param [Integer] section_id
-# @returns [Boolean] true iff the current user is the owner of the given section.
+# @returns [Boolean] true if the current user is the owner of the given section.
 #          Note: NOT always true for admins.
```

Comment:
:laughing: This isn't a typo - it's [shorthand for "if and only if"](https://en.wikipedia.org/wiki/If_and_only_if).

Reply:
But maybe you're right to correct that, given the note on the next line!

Reply:
ah, I had no idea. Thanks!